source/simulator/parameters.cc

```diff
@@ -318,6 +318,30 @@ namespace aspect
                        "the composition system gets solved. See 'linear solver "
                        "tolerance' for more details.");
 
+    prm.enter_subsection ("Solver parameters");
+    {
+      prm.enter_subsection ("Newton solver parameters");
+      {
+        prm.declare_entry ("Nonlinear Newton solver switch tolerance", "1e-5",
+                           Patterns::Double(0,1),
+                           "A relative tolerance with respect to the the residual of the first "
+                           "iteration, up to which the nonlinear Piccard solver will iterate, "
+                           "before changing to the newton solver.");
+
+        prm.declare_entry ("Max pre-Newton nonlinear iterations", "10",
+                           Patterns::Integer (0),
+                           "The maximum number of Piccard nonlinear iterations to be performed "
+                           "before switching to Newton iterations.");
+
+        prm.declare_entry ("Max Newton line search iterations", "5",
+                           Patterns::Integer (0),
+                           "The maximum amount of line search iterations allowed. If the "
```

Comment:
-> maximum *number*

("amount" is the word used for uncountable things, like milk, hay, air; for countable things, use "number")